test/plugin/test_filter_record_transformer.rb

```diff
@@ -83,6 +84,14 @@ def filter(config, msgs = [''])
       end
     end
 
+    test 'remove_keys with nested key' do
+      config = CONFIG + %[remove_keys $.nest.k1]
+      filtered = filter(config)
+      filtered.each_with_index do |(_t, r), i|
+        assert_not_include({'k2' => 'v2'}, r['nest'])
```

Comment:
This line should be `assert_not_include(r['nest'], 'k1')`.

Reply:
right. just fixed.